test/api/v3/integration/members/GET-members_id.test.js

```diff
@@ -41,6 +41,7 @@ describe('GET /members/:memberId', () => {
     expect(memberRes.stats.maxMP).to.exists;
     expect(memberRes.stats.maxHealth).to.equal(common.maxHealth);
     expect(memberRes.stats.toNextLevel).to.equal(common.tnl(memberRes.stats.lvl));
+    expect(memberRes.inbox.optOut).to.exists;
```

Comment:
The problem is that in chai, using a property that doesn't exist does not throw an error. Ie `expect(something).to.foobarbaz` will not error. 

I'm working on adjusting our linter to prevent using properties for assertions.